test/functional/tests/io_class/test_io_class_cli.py

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+
+import pytest
+
+from api.cas import casadm, ioclass_config
+from api.cas.cache_config import CacheMode
+from api.cas.casadm_params import OutputFormat
+from api.cas.ioclass_config import IoClass
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools import fs_utils
+from test_utils.size import Size, Unit
+
+ioclass_config_path = "/tmp/opencas_ioclass.conf"
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrize("cache_mode", CacheMode)
+def test_ioclass_export_configuration(cache_mode):
+    """
+    title: Export IO class configuration to a file
+    description: Test CAS ability to create a properly formatted file with current IO class
```

Comment:
I've recently learned, that multi line strings in yaml should be done as below, using '|' operator:
"""
description: |
  first line
  second line
"""

So please change it in each test's description.